spec/sidekiq/benefits_intake_status_job_spec.rb

```diff
@@ -17,17 +17,6 @@
 
         BenefitsIntakeStatusJob.new.perform
       end
-
-      context 'form submission has one pending attempt and one successful attempt' do
-        it 'does not process the form submission' do
-          form_submission = create(:form_submission)
-          create(:form_submission_attempt, :pending, form_submission:)
-          create(:form_submission_attempt, :vbms, form_submission:)
-          expect_any_instance_of(BenefitsIntake::Service).not_to receive(:bulk_status)
-
-          BenefitsIntakeStatusJob.new.perform
-        end
-      end
```

Comment:
Are we no longer concerned with this scenario?

Reply:
Another thought, are we testing form submissions with multiple attempts now that we've uncovered that wasn't a thing we had before the migrations you ran?

Reply:
Good call, I've added such a test.